core/services/wifi/WifiManager.py

```diff
@@ -106,6 +113,16 @@ def __dict_from_list(data: bytes) -> Dict[str, Any]:
 
         return output
 
+    @property
+    def hotspot(self) -> HotspotManager:
+        if self._hotspot is None:  # pylint: disable=access-member-before-definition
```

Comment:
What is the problem to define `self._hotspot` as None by default in the constructor ?

Reply:
It is. Line 40.

Reply:
It's unnecessary inside a try block group, that's why you need the pylint disable comment, removing from the block will fix it.